cluster/addons/prometheus/prometheus-configmap.yaml

```diff
@@ -0,0 +1,176 @@
+# Prometheus configuration format https://prometheus.io/docs/prometheus/latest/configuration/configuration/
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-config
+  namespace: kube-system 
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: EnsureExists
+data:
+  alerts: |
+    {}
+  prometheus.yml: |
+    rule_files:
+    - /etc/config/rules
```

Comment:
why specify rules and alert files if they are then left empty?

Reply:
Removed